compiler/transform_dialect/gpu/reduction/reduction.mlir

```diff
@@ -0,0 +1,77 @@
+// RUN: iree-run-mlir %s --iree-hal-target-backends=cuda --iree-hal-dump-executable-sources-to=- | head -n -3 | tail -n +2 | \
```

Comment:
Not sure if head/tail will work on Windows.

Reply:
ah yes .. @benvanik do you see a better and portable way?

Reply:
Moving to iree-compile post vacation works without head/tail.